test/make-appointment.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { browser } from 'protractor';
+import {
+  MenuContentPage,
+  AddDoctorPage,
+  ConfirmationPage,
+  AddPatientPage,
+  MakeAppointmentPage
+} from '../src/page';
+import { generateUUID } from '../src/utils/UUID';
+
+describe('Make an appointment', () => {
```

Comment:
It is better if you follow gherking. For example:
Given I want to schedule a meddical appointment (describe on line 11)
and doctor exists in system (it on line 21)
and I create my patient user (it on line 29)
when I schedule the appointment on <Date> 
then I should get a confirmation message 

for the last two steps you should split the **it** function on line 37